modules/installation-requirements-user-infra.adoc

```diff
@@ -197,6 +197,88 @@ On your z/VM instances, set up:
 * 16 GB for the temporary {product-title} bootstrap machine
 endif::ibm-z[]
 
+ifdef::ibm-power[]
+
+[id="minimum-ibm-power-system-requirements_{context}"]
+== Minimum IBM Power Systems requirements
+
+You can install {product-title} version {product-version} on the following IBM hardware:
+
+* IBM POWER8 or POWER9 processor-based systems
+
+[discrete]
+=== Hardware requirements
+
+* 6 IBM Power bare-metal servers or 6 LPARs across multiple PowerVM servers.
+
+[discrete]
+=== Operating system requirements
+
+* One instance of IBM POWER8 or POWER9 processor-based systems
```

Comment:
systems is plural but it says "one instance". Should it be "One instance of an IBM POWER8 or POWER9 processor-based systems." ?